README.md

```diff
@@ -38,7 +38,7 @@
 <!-- PROJECT DESCRIPTION -->
 
 # Live app<a name="live-link"></a>
-** https://paulshaun-kdtk.github.io/Microverse_Proffessional_Portifolio/
+** https://paulshaun-kdtk.github.io/capstone-Car-Show/
 
 # Video explanation of app <a name="video-explanation"> </a>
 
```

Comment:
- [ ] Kindly consider giving appropriate credit to Cindy Shin the author of the original design, as required by the Creative Commons License. You can add this in the `Acknowledgments` section of the README